### PR TITLE
add halo to item WML code

### LINKED ISSUE #7: Refine `WISH ITEM PICKUP` with halo
- Items could use a twinkling halo to catch the eyes of the players
- This is similar to what LotI uses.

```ini
#define WISH_ITEM_HALO
    halo = "halo/misc/leadership-flare-1.png:20,halo/misc/leadership-flare-2.png:20,halo/misc/leadership-flare-3.png:20,halo/misc/leadership-flare-4.png:20,halo/misc/leadership-flare-4.png:20,halo/misc/leadership-flare-6.png:20,halo/misc/leadership-flare-7.png:20,halo/misc/leadership-flare-8.png:20,halo/misc/leadership-flare-9.png:20,halo/misc/leadership-flare-10.png:20,halo/misc/leadership-flare-11.png:20,halo/misc/leadership-flare-12.png:20,halo/misc/leadership-flare-13.png:20,misc/blank-hex.png:1000"
#enddef
```

### PR DESCRIPTION
**Pull request by knyghtmare**
image
- Should have posted a GIF

Closes #7

### CODE REVIEW

File: utils/items.cfg [hunk 1]
```diff
@@ -8,6 +8,10 @@
     [/trait]
 #enddef
 
+#define WISH_ITEM_HALO
+    halo = "halo/misc/leadership-flare-1.png:20,halo/misc/leadership-flare-2.png:20,halo/misc/leadership-flare-3.png:20,halo/misc/leadership-flare-4.png:20,halo/misc/leadership-flare-4.png:20,halo/misc/leadership-flare-6.png:20,halo/misc/leadership-flare-7.png:20,halo/misc/leadership-flare-8.png:20,halo/misc/leadership-flare-9.png:20,halo/misc/leadership-flare-10.png:20,halo/misc/leadership-flare-11.png:20,halo/misc/leadership-flare-12.png:20,halo/misc/leadership-flare-13.png:20,misc/blank-hex.png:1000"
```
Comment (babaissarkar) [inline utils/items.cfg]: the `halo=` part is a duplicate and already specified below.

Comment (babaissarkar) [reply utils/items.cfg]: Could be shortened using the syntax in https://wiki.wesnoth.org/AnimationWML#Progressive_strings.

Comment (babaissarkar) [reply utils/items.cfg:12]: Change:
```
"halo/misc/leadership-flare-[1~13]:[20*13], misc/blank-hex.png:1000"
```

**Review by babaissarkar — CHANGES_REQUESTED** [review]

File: utils/items.cfg (hunk 1 above)
Comment (babaissarkar) [reply utils/items.cfg]: remove the `halo =` and replace the animation string with `halo/misc/leadership-flare-[1~13].png:[20*13],misc/blank-hex.png:1000"`

Comment (knyghtmare) [reply utils/items.cfg]: Done

**Review by babaissarkar — APPROVED** [review]

**Pull request merged**